api_app/analyzers_manager/constants.py

```diff
@@ -2,33 +2,32 @@
 # See the file 'LICENSE' for copying permission.
 
 from enum import Enum
+import typing
 
 
-class TypeChoices(Enum):
-    FILE = "file"
-    OBSERVABLE = "observable"
-
+class AnalyzerConfigEnum(Enum):
```

Comment:
Could you rename this to like `BaseEnum`? Including "Analyzer" in its name is kinda confusing.